extensions/amp-carousel/validator-amp-carousel.protoascii

```diff
@@ -68,3 +67,71 @@ tags: {  # <amp-carousel>
     supported_layouts: RESPONSIVE
   }
 }
+tags: {  # <amp-carousel> lightbox
+  tag_name: "AMP-CAROUSEL"
+  spec_name: "AMP-CAROUSEL[lightbox]"
+  requires_extension: "amp-carousel"
+  requires_extension: "amp-lightbox-gallery"
+  attrs: {
+    name: "arrows"
+    value: ""
+  }
+  attrs: {
+    name: "autoplay"
+    value: ""
+  }
+  attrs: { name: "controls" }
+  attrs: {
+    name: "delay"
+    value_regex: "[0-9]+"
+  }
+  attrs: {
+    name: "dots"
+    value: ""
+  }
+  attrs: {
+    name: "lightbox"
+    mandatory: true
+  }
+  attrs: {
+    name: "loop"
+    value: ""
+  }
+  attrs: {
+    name: "type"
+    value_regex: "slides|carousel"
+  }
+  # <amp-bind>
+  attrs: { name: "[slide]" }
+  attr_lists: "extended-amp-global"
+  spec_url: "https://www.ampproject.org/docs/reference/components/amp-carousel"
+  amp_layout: {
+    supported_layouts: FILL
+    supported_layouts: FIXED
+    supported_layouts: FIXED_HEIGHT
+    supported_layouts: FLEX_ITEM
+    supported_layouts: INTRINSIC
+    supported_layouts: NODISPLAY
+    supported_layouts: RESPONSIVE
+  }
+  reference_points: {
+    tag_spec_name: "AMP-CAROUSEL lightbox [child]"
+  }
+}
+tags: {
+  html_format: AMP
+  html_format: EXPERIMENTAL
+  tag_name: "$REFERENCE_POINT"
+  spec_name : "AMP-CAROUSEL lightbox [child]"
+  attr_lists: "children-of-lightboxed-carousel"
+}
+attr_lists: {
+  name: "children-of-lightboxed-carousel"
```

Comment:
I don't see `children-of-lightboxed-carousel` being reused anywhere. Can it be removed and its `attrs` put inside the `"AMP-CAROUSEL lightbox [child]"` spec?